development.ini.sample

```diff
@@ -46,6 +46,10 @@ port = 8080
 [app:main]
 use = egg:acm-website
 
+# Set site name and short site name
+#site_name =
+#short_site_name =
```

Comment:
use a key like ``site.name`` and ``site.short_name``.

It is not clear what the purpose of name vs short name is. Please document or just use one.

Also, document the behavior when only one is specified.